activemodel/lib/active_model/validations/clusivity.rb

```diff
@@ -24,7 +26,7 @@ def include?(record, value)
           delimiter
         end
 
-        if value.is_a?(Array)
+        if value.is_a?(Array) && options[:all] == true
```

Comment:
The idea behind the `all` name is that we can introduce also an `any` option (use `.any?` instead of `.all?`) which makes sense for the `exclusion` validation, check cases such as: https://github.com/rails/rails/pull/22610#issuecomment-749246114